src/main/java/com/iyzipay/IyzipayResource.java

```diff
@@ -36,6 +37,7 @@ public class IyzipayResource {
     public IyzipayResource() {
     }
 
+    @Deprecated
```

Comment:
There is no explanation provided for why it has been deprecated or what should be used instead. 
Could you add information about it?